Userland/Libraries/LibShell/AST.cpp

```diff
@@ -14,12 +14,15 @@
 #include <AK/StringBuilder.h>
 #include <LibCore/EventLoop.h>
 #include <LibFileSystem/FileSystem.h>
-#include <LibURL/URL.h>
 #include <errno.h>
 #include <fcntl.h>
 #include <signal.h>
 #include <unistd.h>
 
+#ifdef USE_LIBURL
```

Comment:
No way, nu-uh.
Please come up with a better solution, or simply accept the overhead of including those libraries.